odoo/addons/base/migrations/11.0.1.3/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,429 @@
+# DONE: pre-migration: copy column
+# TODO: post-migration: map values 'trigger' and 'client_action' to default value
+
+base         / ir.actions.server        / sub_model_object_field (many2one): DEL relation: ir.model.fields
+base         / ir.actions.server        / sub_object (many2one)         : DEL relation: ir.model
+# DONE: pre-migration: rename column
+
+base         / ir.actions.server        / usage (char)                  : now required, default = function
+base         / ir.actions.server        / usage (char)                  : selection_keys is now '['ir_actions_server', 'ir_cron']' ('False')
+base         / ir.actions.server        / usage (char)                  : type is now 'selection' ('char')
+# DONE: pre-migration: copy column
+# TODO: post-migration: set values to default value
+
+base         / ir.actions.server        / use_create (selection)        : DEL required: required, selection_keys: ['copy_current', 'copy_other', 'new', 'new_other'], req_default: function
+base         / ir.actions.server        / use_relational_model (selection): DEL required: required, selection_keys: ['base', 'relational'], req_default: function
+base         / ir.actions.server        / use_write (selection)         : DEL required: required, selection_keys: ['current', 'expression', 'other'], req_default: function
+base         / ir.actions.server        / wkf_field_id (many2one)       : DEL relation: ir.model.fields
+base         / ir.actions.server        / wkf_model_id (many2one)       : DEL relation: ir.model
+base         / ir.actions.server        / wkf_transition_id (many2one)  : DEL relation: workflow.transition
+base         / ir.actions.server        / write_expression (char)       : DEL 
+# DONE: pre-migration: rename column
+
+base         / ir.actions.todo          / groups_id (many2many)         : DEL relation: res.groups
+# NOTHING TO DO
+
+base         / ir.actions.todo          / note (text)                   : DEL 
+base         / ir.actions.todo          / type (selection)              : DEL required: required, selection_keys: ['automatic', 'manual', 'once'], req_default: function
+# DONE: pre-migration: rename column
+
+base         / ir.attachment            / access_token (char)           : NEW 
+# NOTHING TO DO
+
+base         / ir.config_parameter      / group_ids (many2many)         : DEL relation: res.groups
+base         / ir.cron                  / _inherits (False)             : NEW 
+# NOTHING TO DO
+
+base         / ir.cron                  / args (text)                   : DEL 
+base         / ir.cron                  / function (char)               : DEL 
+# DONE: pre-migration: rename column
+
+base         / ir.cron                  / interval_type (selection)     : selection_keys is now '['days', 'hours', 'minutes', 'months', 'weeks']' ('['days', 'hours', 'minutes', 'months', 'weeks', 'work_days']')
+# DONE: pre-migration: copy column
+# TODO: post-migration: map value 'work_days' to default value
```

Comment:
How about mapping 'work_days' to 'days', as the default value is 'months'?

Reply:
Updated todo text.